assembly/std/arguments.ts

```diff
@@ -0,0 +1,165 @@
+import {Address} from './address';
+import {ByteArray} from '@massalabs/as/assembly/byteArray';
+
+/**
+ * Args for remote function call
+ */
+export class Args {
+  private offset: i64 = 0;
+  private argsString: string = '';
+
+  /**
+   *
+   * @param {string} argsString
+   * @param {Array<string>} types
+   */
+  constructor(argsString: string = '') {
+    this.argsString = argsString;
+  }
+
+  /**
+   *
+   * @return {string} the serialized string
+   */
+  serialize(): string {
+    return this.argsString;
+  }
+
+  // getters
+
+  /**
+   *
+   * @return {Address} the address
+   */
+  nextAddress(): Address {
+    const address = new Address();
+    this.offset = address.fromStringSegment(
```

Comment:
You are enforcing a bad practice: serialization/deserialization from/to string. The good practice is to use byte arrays to do so.

It will not be possible to add all objects here, so you should consider this function as a documented template on how to deserialize any object from Args. And therefore, document this function as such.

Reply:
I've created an issue #35